docs/content/preview/api/ysql/the-sql-language/statements/cmd_copy.md

```diff
@@ -152,3 +152,8 @@ yugabyte=# COPY large_table FROM '/home/yuga/Desktop/large_table.csv'
 - If the table does not exist, errors are raised.
 - `COPY TO` can only be used with regular tables.
 - `COPY FROM` can be used with tables, foreign tables, and views.
+
+Additionally, the following copy options may be useful for speeding up copying, or allowing for faster recovery from a partial state:
+* `DISABLE_FK_CHECK` will skip foreign key check when copying new rows to the table.
+* `REPLACE`: if the primary / unique key of the new row conflicts with that of the existing row, the new row will replace the existing row in the table.
+* `SKIP n`: skips the first `n` rows of the file. Require `n` to be nonnegative.
```

Comment:
I would prefer to put a subject before `Require` to be more "grammarly" clear in English. Either change it to `, and requires...` or `n is required to...`. But the specific English wording is up to you as I am not a native speaker either.

Also, should we put more detailed behaviors for `SKIP` here? An example is that it aborts the copy operation when a skipped row in the file is il-formated.

Reply:
https://phabricator.dev.yugabyte.com/D17311 will fix this behavior (e.g. allowing for skipping invalid rows).